btcjson/chainsvrcmds_test.go

```diff
@@ -200,6 +200,18 @@ func TestChainSvrCmds(t *testing.T) {
 			unmarshalled: &btcjson.GetBlockHashCmd{Index: 123},
 		},
 		{
+			name: "getblockheader",
+			newCmd: func() (interface{}, error) {
+				return btcjson.NewCmd("getblockheader", "blockHash")
```

Comment:
Can you make the hash string "123" so it more closely matches `getblock` and is a valid hash?